src/buf.yaml

```diff
@@ -11,17 +11,6 @@ version: v1
 breaking:
   use:
     - WIRE
-  ignore:
-    # does currently not require backward-compatibility
-    - cluster-client
-    # does currently not require backward-compatibility
-    - compute-client
-    # does currently not require backward-compatibility
-    - storage-client
-    # does not currently require backward-compatibility
-    - stash
```

Comment:
This is still true; `objects.proto` can evolve in backwards-incompatible ways because we actively rewrite the stash using migrations.

Reply:
I was under the impression that we always make new proto files. It looks like the exception should only exist for `objects.proto` and all the `_v123` suffixed files should be part of the check. Does this sound right?

Reply:
Yes, or even stricter: `objects.proto` is allowed to evolve, the `_v123` suffixed files should never change, even in backwards-compatible ways.